### DISCUSSION TITLE
Has anyone ever integrated Ignite with the NX mono-repo management tooling?

### DISCUSSION THREAD

**Discussion by duro** [infinitered/ignite#1813] [Q&A] [answered]
In the past I have used the [NX build framework](https://nx.dev/), which provides awesome tooling for building and managing mono-repos.

I'm curious if anyone has had luck using the Ignite boilerplate along with NX?

**Comment by jamonholmgren** [comment]
I haven't! Looks interesting, though!

**Reply by jamonholmgren** [reply]
There is this: https://blog.nrwl.io/introducing-react-native-support-for-nx-48d335e90c89

**Reply by jamonholmgren** [reply]
@duro

**Reply by trumbitta** [reply]
Also Nx 13 has now official support for RN (it was an unofficial plugin before) 🚀

**Comment by glesperance** [comment]
Any updates on this / is there any documentation one could be pointed to? I'm trying to move my ignite-cli app to a Nx monorepo but with no luck

**Comment by donnes** [comment]
Hi @glesperance! Maybe you can give a chance for the [Vercel's Turborepo](https://turborepo.org/).
I have being working in a monorepo using this tool and it's simply amazing, fast and easy to scale!

**Reply by glesperance** [reply]
Yeah after hours of trying to make Nx work without success, I just switched to Turbo repo and got started instantly.

**Comment by jamonholmgren** [comment, marked as answer]
Official support for React Native announced here: https://blog.nrwl.io/core-performance-improvements-webpack-5-react-native-and-more-in-nx-13-cfe63c7415f1

**Reply by Sowed** [reply]
Has anyone had any chance using `nx` with an existing `ignite` project? The nx documentation shows support for react native, but ...
- The bootstrapped project with the react native plugin does not use expo modules.
- The one bootstrapped with the expo plugin is a managed expo app.

I have failed to find a way of moving my ignite cli generated react native app to nx.
- I tried to add expo modules to an app generated by the nx react-native plugin but still in vain.
- I do not wish to use the nx expo plugin app and eject from that.

Good people, has anyone had luck moving an existing ignite cli app into an nx workspace? Any help will be appreciated.
At this point, I am contemplating moving to turborepo but I had a great project going with nx already 😮‍💨
Thank you.

**Comment by glesperance** [comment]
I just went with turborepo. It was easy enough!
